featuretools/tests/requirement_files/minimum_test_requirements.txt

```diff
@@ -11,3 +11,4 @@ boto3==1.17.46
 composeml==0.3.0
 urllib3==1.26.5
 pyarrow==3.0.0
+holidays>=0.12
```

Comment:
Hmm...I don't think you should need this here. These should just be packages that are needed for running the tests, but not already included in the other requirements files .

Reply:
![CleanShot 2022-01-19 at 10 45 15](https://user-images.githubusercontent.com/2358574/150165405-45ec88ff-ffed-4eaa-b958-14d3dee33dd7.png)

This test was failing

Reply:
last line does a pip instal from `minimum_test_requirements.txt`

Reply:
I think the holidays install for these tests should get handled a little lower in this file. It will try to install requirements from `minimum_core_requirements.txt` or `minimum_koalas_requirements.txt` depending on the version of the test being run, so as long as you have `holidays` included in those two files I think you should be good.

![image](https://user-images.githubusercontent.com/4307001/150166479-a1f3ed1e-b8aa-46a9-8069-17da46ba1219.png)

Reply:
One other point of clarification here, in the `minimum_*` files, we will want to pin the version to a specific number to make sure we always get that lowest version installed, instead of using `>=version` syntax which would allow newer versions to be installed.